operator/v1/types_dns.go

```diff
@@ -30,6 +30,45 @@ type DNS struct {
 
 // DNSSpec is the specification of the desired behavior of the DNS.
 type DNSSpec struct {
+	// servers is a list of DNS resolvers that provide name query delegation for one or
+	// more subdomains outside the scope of the cluster domain. If Servers consists of
+	// more than one Server, longest suffix match will be used to determine the Server.
+	//
+	// For example, if there are two Servers, one for "foo.com" and another for "a.foo.com",
+	// and the name query is for "www.a.foo.com", it will be routed to the Server with Zone
+	// "a.foo.com".
+	//
+	// If this field is nil, no servers are created.
+	//
+	// +optional
+	Servers []Server `json:"servers,omitempty"`
+}
+
+// Server defines the schema for a server that runs per instance of CoreDNS.
+type Server struct {
+	// name is required and specifies a	unique name for the server. Name must comply
+	// with the Service Name Syntax of rfc6335.
+	Name string `json:"name"`
+	// zones is required and specifies the subdomains that Server is authoritative for.
+	// Zones must conform to the rfc1123 definition of a subdomain. The cluster domain
+	// (i.e. cluster.local) is an invalid Zones subdomain.
```

Comment:
"Zones" → "zones", or reword as, "Specifying the cluster domain (i.e., "cluster.local") is invalid."